test/parallel/test-worker-eventlooputil.js

```diff
@@ -68,8 +69,8 @@ let workerELU;
 
 
 function checkWorkerIdle(wElu) {
-  const tmpMainElu = eventLoopUtilization(mainElu);
   const perfWorkerElu = workerELU();
+  const tmpMainElu = eventLoopUtilization(mainElu);
   const eluDiff = eventLoopUtilization(perfWorkerElu, mainElu);
```

Comment:
Completely remove the `eluDiff` test. I don't remember what I was thinking when I added this, but the syntax is invalid. The diff between different ELU's is only valid for the same thread. This does a diff between worker and parent.

Reply:
done.

We could add `worker.threadId` into the elu result and throw on mismatch as it is a clear wrong use.